### PR TITLE
feat(noValidate): support auro-input noValidate attribute #37

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #37

## Summary:

Add the `noValidate` attribute which is passed down to auro-input.

Also adds documentation examples for the attribute.

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by MattSidor — DISMISSED** [review]

File: demo/apiExamples.md [hunk 1]
```diff
@@ -8,12 +8,19 @@
...
 | [required](#required)       | `required`       | `Boolean` |         | Populates the `required` attribute on the input. Used for client-side validation. |
 | [triggerIcon](#triggerIcon)    | `triggerIcon`    | `Boolean` |         | If set, the `icon` attribute will be applied to the trigger `auro-input` element. |
 | [type](#type)           | `type`           | `String`  |         | Applies the defined value as the type attribute on auro-input. |
 | [value](#value)          | `value`          | `String`  | null    | Value selected for the dropdown menu.            |
 
+## Events
+
+| Event                | Type               |
+|----------------------|--------------------|
+| `auroCombobox-ready` | `CustomEvent<any>` |
```
Comment (blackfalcon) [inline demo/apiExamples.md:22]: I think this addition to the API docs was a miss from the last updates as I am not seeing this code added here? Please update the reference with an `@fires` identifier and describe the event.

File: demo/apiExamples.md [hunk 2]
```diff
@@ -138,6 +145,44 @@ Sets a persistent error state (e.g. an error state returned from the server).
 
 </auro-accordion>
 
+#### noValidate
+
+Intended for use with the `required` attribute. If set, disables auto-validation on blur. By using these two attributes in combination the validation for required fields is still computed for forms but no validation messaging will be generated in the UI.
```
Comment (blackfalcon) [inline demo/apiExamples.md:150]:
```suggestion
Intended for use with the `required` attribute. When set, it disables `auto-validation` on blur. When using these two attributes in combination the validation for required fields is still computed for forms but no validation messaging will be generated in the UI.
```

File: docs/partials/apiExamples.md [hunk 3]
```diff
@@ -53,6 +53,22 @@ Sets a persistent error state (e.g. an error state returned from the server).
 
 </auro-accordion>
 
+#### noValidate
+
+Intended for use with the `required` attribute. If set, disables auto-validation on blur. By using these two attributes in combination the validation for required fields is still computed for forms but no validation messaging will be generated in the UI.
```
Comment (blackfalcon) [inline docs/partials/apiExamples.md:58]:
```suggestion
Intended for use with the `required` attribute. When set, it disables `auto-validation` on blur. When using these two attributes in combination the validation for required fields is still computed for forms but no validation messaging will be generated in the UI.
```

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: demo/apiExamples.md (hunk 2 above)
Comment (jason-capsule42) [reply demo/apiExamples.md:150]: Language used here is cut and paste from the language in auro-input description of this attribute. I think we should keep these in sync.

File: docs/partials/apiExamples.md (hunk 3 above)
Comment (jason-capsule42) [reply docs/partials/apiExamples.md:58]: Same comment as above.

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**